### PR TITLE
JENKINS-56109 Make traits.jelly compatible with table => div

### PR DESCRIPTION
**Pull request by timja**
ref: https://github.com/jenkinsci/jenkins/pull/3895

especially: https://github.com/jenkinsci/jenkins/pull/3895#issuecomment-624285546

Also included a pom update (separate commit) due to recommendedConfigurations change, this will fix: https://github.com/jenkinsci/scm-api-plugin/pull/80

<details>
<summary>Before</summary>

image

image

</details>

<details>
<summary>After</summary>

image

image

</details>

### CODE REVIEW

**Comment by timja** [comment]
@bitwiseman

File: pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.43</version>
+    <version>4.0</version>
```
Comment (bitwiseman) [inline pom.xml:32]:
```suggestion
    <version>4.1</version>
```

File: pom.xml [hunk 2]
```diff
@@ -67,9 +67,10 @@
   <properties>
     <revision>2.6.4</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.107.3</jenkins.version>
+    <jenkins.version>2.190.3</jenkins.version>
```
Comment (bitwiseman) [inline pom.xml:70]:
```suggestion
    <jenkins.version>2.176.4</jenkins.version>
```

File: pom.xml [hunk 3]
```diff
@@ -85,28 +86,31 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.190.x</artifactId>
```
Comment (bitwiseman) [inline pom.xml:93]:
```suggestion
        <artifactId>bom-2.176.x</artifactId>
```

**Comment by timja** [comment]
The reason I picked 2.190 is to drop the trilead-api implied dependency but happy to change it

**Comment by timja** [comment]
Thoughts @bitwiseman ?

**Comment by timja** [comment]
ping @bitwiseman @dwnusbaum

**Comment by timja** [comment]
@dwnusbaum added screenshots, no changes to the UI at all

**Comment by dwnusbaum** [comment]
@timja Thanks!

**Pull request merged**